tensorflow/core/grappler/optimizers/auto_mixed_precision.cc

```diff
@@ -2298,11 +2311,19 @@ Status AutoMixedPrecision::Optimize(Cluster* cluster, const GrapplerItem& item,
   *output = item.graph;
 
   int num_gpus = GetNumGPUs(*cluster);
+  bool run_fp16_on_cpu = false;
   if (num_gpus < 1 && mode_ == AutoMixedPrecisionMode::CUDA) {
-    // AutoMixedPrecision is currently only tuned for GPU.
-    LOG(WARNING) << "No (suitable) GPUs detected, skipping " << name()
-                 << " graph optimizer";
-    return OkStatus();
+    // No GPUs to run AutoMixedPrecision in FP16.
+    // Check if CPU supports
```

Comment:
nit: finish sentence, use proper grammar and punctuation.

Reply:
Fixed